prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -28,6 +28,7 @@ import gov.cdc.prime.router.tokens.TokenAuthentication
 import org.apache.logging.log4j.kotlin.Logging
 
 private const val CLIENT_PARAMETER = "client"
+private const val PAYLOAD_NAME_HEADER = "payloadname" // Headers get normalized to lowercase
 private const val PAYLOAD_NAME_PARAMETER = "payloadName"
```

Comment:
It seems like the cgi parameter and header should be the same?

Reply:
I'd thought we had camelCase cgi params already but we only had one-word params! Updated. 👍🏼